lib/open_api_spex/plug/put_api_spec.ex

```diff
@@ -48,4 +44,28 @@ defmodule OpenApiSpex.Plug.PutApiSpec do
     |> Stream.map(fn operation -> {operation.operationId, operation} end)
     |> Enum.into(%{})
   end
-end
\ No newline at end of file
+
+  defp fetch_spec(spec_module) do
+    # This if statement and the contents of the else block can be
+    # removed once an Elixir lang release requires an OTP version >=21.2
+    if function_exported?(:persistent_term, :info, 0) do
+      try do
+        :persistent_term.get(spec_module)
+      rescue
+        ArgumentError ->
+          IO.puts("Warning: OpenApiSpec spec not found in persistent terms")
```

Comment:
`IO.warn/1` might be better for this.

Reply:
The only issue with that is this will always occur one time on the first call since it won't exist yet. I had it as a warning originally. I might have a workaround to deal with that.